dozer-types/src/types/field.rs

```diff
@@ -44,6 +48,7 @@ impl ToPyObject for Field {
             }
             Field::Bson(val) => val.to_object(py),
             Field::Null => unreachable!(),
+            Field::Point(_val) => todo!(),
```

Comment:
@xudong963 can you check this and suggest what to do?

Reply:
Leave it as todo, after the ticket merged, I'll open a ticket to complete it.